opensearch_py_ml/ml_models/sentencetransformermodel.py

```diff
@@ -791,6 +791,13 @@ def save_as_pt(
             zip_file_name = str(model_id.split("/")[-1] + ".zip")
         zip_file_path = os.path.join(model_output_path, zip_file_name)
 
+        # handle undefined model_max_length in model's tokenizer (e.g. "intfloat/e5-small-v2" )
+        if model.tokenizer.model_max_length == 1000000000000000019884624838656:
```

Comment:
Let's assign a static variable for this magic number and also add more comments including the github issue why we need this. 

In addition, I assume this large is same for every cases, but I was wondering if we should do something like:

model.tokenizer.model_max_length = min (model.tokenizer.model_max_length, model.get_max_seq_length())

Can you please check if that's applicable for every cases?

Reply:
I just made some changes. I use `if model.tokenizer.model_max_lengt > model.get_max_seq_length()` instead of `min` function, so that we can add `print()` to let user know what's happening.